pendulum_msgs/msg/JointState.msg

```diff
@@ -0,0 +1,3 @@
+float64 position
```

Comment:
I agree that only using static messages for realtime is a huge limitation and I should at least support dynamically sized messages with a bounded size. I actually started a pull request related to this but I put it on the backburner to focus on what the demo would look like.

I think Adolfo is suggesting a way to lock dynamically sized messages during the startup stage of the node lifecycle, so that you can use a preallocated, statically sized buffer for the message during execution. Is that accurate? It seems like this is useful when the size of the messages becomes known when the program starts, but does not change during real-time execution.